packages/components/src/internal/components/lineage/vis/VisGraph.tsx

```diff
@@ -176,26 +176,24 @@ export class VisGraph extends Component<VisGraphProps, VisGraphState> {
     };
 
     fitGraph = (): void => {
-        // NK: This is a arbitrarily set heuristic for deciding when to focus vs when to fit to
+        const { seed } = this.props;
+
+        // NK: This is an arbitrarily set heuristic for deciding when to focus vs when to fit to
         // the entire graph. I'm trying to delineate "small" graphs from "large" graphs.
-        if (this.network.body.nodeIndices.length > 50) {
-            this.focusSeed();
+        if (this.network.body.nodeIndices.length > 50 && seed) {
+            // Focus on the seed (if set)
+            this.network.fit({ animation: false, nodes: [seed] });
+
+            // Zoom out a bit from the default fitted zoom level
+            const scale = this.network.getScale() - 0.25;
+            if (scale > 0) {
+                this.network.moveTo({ scale });
+            }
         } else {
             this.network.fit();
         }
     };
 
-    focusSeed = (): void => {
-        // Focus on the seed
-        this.network.fit({ animation: false, nodes: [this.props.seed] });
-
-        // Zoom out a bit from the default fitted zoom level
-        const scale = this.network.getScale() - 0.25;
-        if (scale > 0) {
-            this.network.moveTo({ scale });
-        }
-    };
-
```

Comment:
the changes in this file seem unrelated to the issue 51853. Was there another issue fix here or was this just something you noticed locally and were fixing? (just curious if another issue comment is needed here for this change).

Reply:
> This also addresses an issue reported by a user where when the lineage graph attempts to focus on the seed node it can cause the app to crash. I wasn't able to reproduce the scenario but am proactively adding defensiveness so that we prevent calling into the focusing if a seed value is not provided.

I put a comment in the PR description.

Reply:
[Ticket 51826](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=51826)

Reply:
Sounds good. Thanks. I missed that comment.